domainconnectzone/qsutil.py

```diff
@@ -1,4 +1,4 @@
-def qs2dict(qs, filter=[]):
+def qs2dict(qs, filter_items=[]):
```

Comment:
Changed this name to suppress warnings about shadowing the Python reserved word `filter`.